script/start_server.sh.sample

```diff
@@ -50,6 +50,19 @@ if ! command -v $PUMA_CMD 2>&1 >/dev/null; then
   exit 1
 fi
 
+# Pick a system stack:
+# UNCOMMENT AS REQUIRED ON NON SAMPLE FILE
+# STACK="beta"
+# STACK="live"
+
+if [ -n "$STACK" ]; then
```

Comment:
Can we please add `STACK` to the table in the usage comments at the top of this file please?

Reply:
updated